build/CodeAnalysis.ruleset

```diff
@@ -1,12 +1,9 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <RuleSet Name="Production Assembly Rules" ToolsVersion="15.0">
-	<Include Path="CargoWiseRuleSet.Common.ruleset" Action="Default" />
 	<Include Path="StyleCopAnalyzers.ruleset" Action="Default" />
 
-	<Rules AnalyzerId="Microsoft.Analyzers.ManagedCodeAnalysis" RuleNamespace="Microsoft.Rules.Managed">
-		<Rule Id="CA1014" Action="None" />
-		<Rule Id="CA1020" Action="None" />
-		<Rule Id="CA1811" Action="None" />
+	<Rules AnalyzerId="Microsoft.CodeQuality.Analyzers" RuleNamespace="Microsoft.CodeQuality.Analyzers">
+		<Rule Id="CA1506" Action="Info" />
```

Comment:
Which classes triggered this?

Reply:
I don't recall, I'm sure it was in one of the test assemblies but can't get it to happen again ... removed.